src/utils.ts

```diff
@@ -83,3 +83,26 @@ export function debounceAsync<R, F extends (...args:any[]) => PromiseLike<R>>(in
 		);
 	} as F;
 }
+
+
+export function makePseudoObservable<T>(initialValue:T) {
+    const _atom = new Atom();
+    let _isSet = false;
+    let _value = initialValue;
+
+    return {
+        get isSet() {
+            _atom.reportObserved();
+            return _isSet;
+        },
+        get value() {
+            _atom.reportObserved();
+            return _value;
+        },
+        set value(v:T) {
+            _value = v;
+            _isSet = true;
+            _atom.reportChanged();
+        }
```

Comment:
I'm not sure why mobx doesn't allow setting observables from inside computed's, I'm assuming they set up their observables basically the same way as I am here